src/satipc.c

```diff
@@ -401,13 +401,14 @@ int satipc_open_device(adapter *ad)
 	if ((sip->last_connect > 0) && (ctime - sip->last_connect < 5000))
 		return 3;
 
+	USockAddr sv;
```

Comment:
What is the problem you are teying to solve by adding SV?

Reply:
The problem of the SAT>IP "client" module not printing the address of the server. Please, note that here the `sv` is used in the next `tcp_connect_src()` call. Passing one empty address will disable the option of logging messages with the full addresses info: local+port and remote+port.

Reply:
Please see the next message why adding one destination to multiple sockets is a bad idea.